pkg/nat/sort.go

```diff
@@ -26,6 +26,7 @@ func (s *portSorter) Less(i, j int) bool {
 	return s.by(ip, jp)
 }
 
+// Sort sorts a list of ports using the provided predicate
 func Sort(ports []Port, predicate func(i, j Port) bool) {
```

Comment:
What invariant should be enforced by `predicate`? `sort.Sort` isn't necessarily stable, so `predicate` _must_ provide a total ordering over `ports` (I think, so doublecheck).

Unrelated to PR: this approach is not very idiomatic Go.